docs/src/description/constraints.md

```diff
@@ -2,19 +2,17 @@
 
 ## Boundary constraints (`boundary_constraints`)
 
-The `boundary_constraints` section consists of expressions describing the expected value of columns in the main or auxiliary traces at the specified boundary. Column boundaries can be selected using boundary accessors. Valid boundary accessors are `.first`, which selects the first cell of the column to which it is applied, and `.last`, which selects the last cell of the column column to which it is applied.
+The `boundary_constraints` section consists of expressions describing the expected value of columns in the main trace at the specified boundary. Column boundaries can be selected using boundary accessors. Valid boundary accessors are `.first`, which selects the first cell of the column to which it is applied, and `.last`, which selects the last cell of the column to which it is applied.
```

Comment:
Boundary constraints can also specify constraints for busses, right? I think we should mention this here.